src/cookie-manager.ts

```diff
@@ -27,9 +27,10 @@ export class CookieManager implements ICookieManager {
     })
   }
 
-  public remove(name: string): void {
+  public remove(name: string): string {
```

Comment:
"remove" method doesn't need to return something because it just removes the cookie.

Reply:
I see